src/core/lib/slice/slice_utils.h

```diff
@@ -150,6 +154,15 @@ struct ExternallyManagedSlice : public UnmanagedMemorySlice {
     data.refcounted.length = length;
     data.refcounted.bytes = bytes;
   }
+  bool Equals(const grpc_slice& other) {
```

Comment:
There is absolutely no convention in C++ that `==` indicates that the two operands are exactly the same object.  In fact, there can't be, because it's entirely possible (and often useful) to define `operator==()` with different types for the two operands.

We define `operator==()` in many plances that checks for field equality, and I think it's fine to do that here as well.

Reply:
Defined as operator==().